src/main/java/org/prebid/server/bidder/beachfront/BeachfrontBidder.java

```diff
@@ -213,27 +199,23 @@ private static ExtImpBeachfront makeVideoImpsAndGetExtImpBeachfront(BidRequest b
     /**
      * Creates {@link BeachfrontVideoRequest} domain from {@link App} or {@link Site}.
      */
-    private static void populateVideoRequestDomainAndSite(App app, Site site,
-                                                          BeachfrontVideoRequest.BeachfrontVideoRequestBuilder
+    private static void populateVideoRequestSitePageAndDomain(App app, Site site,
```

Comment:
i'd like to name the method as `populateVideoRequestSite` to rid off the details.